e2e/keywords/common.resource

```diff
@@ -0,0 +1,42 @@
+*** Settings ***
+Documentation     
+...               Keywords defined here can be used where this Keywords.resource in loaded.
+Library    ../libs/steps.py
+Resource    ../keywords/utility.resource
+
+*** Variable ***
+${retry_time_out}=    600
+${retry_interval}=    5
+
+*** Keywords ***
+Volume state should be ${volume_expect_state}
+    Run keyword And Continue On Failure    Wait Until Keyword Succeeds    ${retry_time_out} seconds    ${retry_interval} seconds    Check volume state should be ${volume_expect_state}
+
+Node ${node_number} with replica node state should be ${node_expect_state}
```

Comment:
What does the node_number here mean?

Reply:
For scenarios where the number of volume replicas is less than that of working nodes, for example, start 2 replicas among 3 working nodes.

In my original thought, since I couldn't assign replica locations, I created two lists to hold nodes with and without replicas. Then in different test cases, the nodes with or without replicas will be abnormal like `Restart Kubelet`: `Scenario-5` and `Scenario-6`.
![image](https://github.com/longhorn/longhorn-tests/assets/17548901/d73d0900-019a-4153-8f3a-b821b19bade1)
https://github.com/longhorn/longhorn-tests/blob/65d41169c11e35b2cdc5a25d83d54ea259ad7dd5/e2e/tests/node_not_ready/restart_kubelet.robot#L82-L110

Therefore, "${node_number}" means the node index with or without replicas, and a description like the following will appear:
 - **Node ${node_number} with replica node** state should be ${node_expect_state}
 - **Node ${node_number} without replica node** state should be ${node_expect_state}

Reply:
I see. Then I suggest using ${node_index} instead to improve clarity.

And also we should avoid duplicating keywords unnecessarily just for the purpose to have different names.
Using `Then Node 1 with replica node state should be Ready` as an example,
you can take this approach instead:
```
Then Node 1 should have 1 volume replica
And Node 1 state should be Ready

And Node 2 should have 0 volume replica
And Node 2 state should be...
```

Reply:
Got it. That's indeed a good way to make the description clearly. Thanks for the suggestion. I will try to do it.

Reply:
I've updated it to consolidate the `node_index` related description without duplicating keywords unnecessarily like below:
https://github.com/longhorn/longhorn-tests/blob/15ca9dcba0534243fb68b9d0060173eafd129a5f/e2e/tests/node_not_ready/node_power_off.robot#L26-L33